travis/tests-unit.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 #set -e
 
-if [ $TRAVIS_PULL_REQUEST == "false" ]; then
+UNIT_TESTS_ALWAYS=1
```

Comment:
@tomkinsc This fixes the issue of coveralls coverage drop, at the cost of again doing redundant computations on travis.  Maybe it could stay this way, till we figure out how to avoid redundant computation without causing incorrect coverage measurements?  @yesimon ?